README.md

```diff
@@ -1,13 +1,12 @@
-[![Build Status master](https://travis-ci.org/ATIX-AG/foreman_scc_manager.svg?branch=master)](https://travis-ci.org/ATIX-AG/foreman_scc_manager)
+![Rubocop](https://github.com/ATIX-AG/foreman_scc_manager/actions/workflows/rubocop.yaml/badge.svg)
```

Comment:
Either add the newly added pipelines from #74 or display an aggregated status, if github provides it.

Reply:
Added the second badge.